seqr/views/apis/anvil_workspace_api.py

```diff
@@ -3,6 +3,11 @@
 import time
 import tempfile
 from datetime import datetime
+import requests
+import base64
```

Comment:
I don't think this is used

Reply:
`requests` is used in `_make_airflow_api_request`, and base64 is not used